tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@tsconfig/node16/tsconfig.json",
+  "compilerOptions": {
+    "resolveJsonModule": true
```

Comment:
Wondering if there's a way we can remove this `tsconfig` file (since there is already one in `/webview`) and still have the unit tests import json...

Reply:
Just move the samples into the webview directory or avoid importing the json module.
In the TS SDK I allow importing a JSON file outside of the source directory but that's not optimal.
The other option is to read the JSON file and use JSON.parse.

https://github.com/temporalio/sdk-typescript/blob/main/packages/worker/src/pkg.ts

Reply:
> Just move the samples into the webview directory 

I've tried this and it still errors, likely because when we compile it for `test` it's still looking at the root level for that `tsconfig` file. Will continue to investigate...

Reply:
**Update:** Fixed this by specifying an alternative path for an env variable in the `test` command (see https://github.com/temporalio/vscode-debugger-plugin/pull/28/commits/326142b48723d76156e32d0a23190e4e8c2781c8).